src/notebook/Events.jl

```diff
@@ -45,6 +45,12 @@ function try_event_call(session, event::PlutoEvent)
     end
 end
 
+
+# Triggered when the web server gets started
+struct ServerStartEvent <: PlutoEvent
+    address::String
```

Comment:
I only include the unparsed address because
1. Everyone's needs are different and this includes the necessary information for connecting to the server
2. There are several different ways to specify the same network configuration

Reply:
I vote for just giving the port as integer as a second field

Reply:
agree - the address that Pluto knows is not useful outside the machine either way (binding to `0.0.0.0:1234` means `<public_ip>:1234` for anyone outside)

Reply:
Good point, I have added in the second field for port